src/routes/ExercisesScreens.tsx

```diff
@@ -1,118 +1,80 @@
-    alignItems: 'center',
-    width: '100%',
-    backgroundColor: COLORS.lunesBlack,
-    borderColor: COLORS.white,
-    borderWidth: 1,
-    borderStyle: 'solid',
-    borderRadius: 2
-  },
-  clickedItemTitle: {
-    textAlign: 'left',
-    fontSize: wp('4.5%'),
-    fontWeight: '600',
-    letterSpacing: 0.11,
-    marginBottom: 2,
-    color: COLORS.lunesWhite,
-    fontFamily: 'SourceSansPro-SemiBold'
-  },
-  clickedItemDescription: {
-    fontSize: wp('4%'),
-    fontWeight: 'normal',
-    color: COLORS.white,
-    fontFamily: 'SourceSansPro-Regular'
-  },
-  level: {
-    marginTop: 11
-  },
-  title2: {
-    textAlign: 'left',
-    fontSize: wp('4.5%'),
-    fontWeight: '600',
-    letterSpacing: 0.11,
-    marginBottom: 2,
-    color: COLORS.lunesGreyDark,
-    fontFamily: 'SourceSansPro-SemiBold'
-  },
-  title: {
-    color: COLORS.lunesBlack,
-    fontFamily: 'SourceSansPro-SemiBold',
-    fontSize: wp('4%'),
-    textTransform: 'uppercase',
-    fontWeight: '600',
-    marginLeft: 15
-  }
-})
+const Root = styled.View`
+  background-color: ${prop => prop.theme.colors.lunesWhite};
+  height: 100%;
+  padding-top: 10%;
+`
+const ItemTitle = styled(FlatList as new () => FlatList<ExerciseType>)`
+  width: ${wp('100%')}px;
+  padding-right: ${wp('5%')}px;
+  padding-left: ${wp('5%')}px;
+`
+
+const ScreenDescription = styled.Text`
+  font-size: ${wp('4%')}px;
+  color: ${props => props.theme.colors.lunesGreyMedium};
+  font-family: ${props => props.theme.fonts.contentFontRegular};
+`
+const Description = styled.Text<{ selected: boolean }>`
+  font-size: ${wp('4%')}px;
+  font-family: ${props => props.theme.fonts.contentFontRegular};
+  font-weight: normal;
+  color: ${props => (props.selected ? props.theme.colors.white : props.theme.colors.lunesGreyDark)};
+`
+const ScreenTitle = styled.Text`
+  text-align: center;
+  font-size: ${wp('5%')}px;
+  color: ${props => props.theme.colors.lunesGreyDark};
+  font-family: ${props => props.theme.fonts.contentFontBold};
+`
+const Container = styled.Pressable<{ selected: boolean }>`
+  align-self: center;
+  padding: 17px 8px 17px 16px;
+  margin-bottom: 8px;
+  flex-direction: row;
+  align-items: center;
+  width: 100%;
+  justify-content: space-between;
+  border-width: 1px;
+  border-style: solid;
+  border-radius: 2px;
+
+  background-color: ${props => (props.selected ? props.theme.colors.lunesBlack : props.theme.colors.white)};
+  border-color: ${props => (props.selected ? props.theme.colors.white : props.theme.colors.lunesBlackUltralight)};
+`
+const StyledItemTitle = styled.Text<{ selected: boolean }>`
+  text-align: left;
+  font-size: ${wp('4.5%')}px;
+  font-weight: 600;
+  letter-spacing: 0.11px;
+  margin-bottom: 2px;
+  font-family: ${props => props.theme.fonts.contentFontBold};
+
+  color: ${props => (props.selected ? props.theme.colors.lunesWhite : props.theme.colors.lunesGreyDark)};
+`
 
-LogBox.ignoreLogs(['Non-serializable values were found in the navigation state'])
```

Comment:
Think it would be good to put this in again.

Reply:
I told her to remove it, as the warning did not appear for me. Did you have this log?

Reply:
Yes, for me it appeared and git-blame showed me Steffen added. But never mind, we can readd it any time, if needed